src/applications/vaos/tests/mocks/helpers.v2.js

```diff
@@ -119,7 +119,7 @@ export function mockVAOSParentSites(ids, data, children = false) {
  * @export
  * @param {Object} params
  * @param {VAOSAppointment} params.appointment Request object from the vaos service that will be returned back
- *    from the mock with the status set to Cancelled and cancelation reason set to patient
+ *    from the mock with the status set to Cancelled
```

Comment:
just a newbie question - how did you know this needed updating?

Reply:
Good question. It didn't need any update at all.  I removed part of the comment by accident.

Reply:
no worries - I went ahead and approved